lms/static/sass/theme-overrides.scss

```diff
@@ -1,93 +1,193 @@
+                color: $user-dropdown-color !important;
+            }
+        }
+    }
+    .main .nav-tab a, .secondary .nav-item a, .secondary .nav-item .toggle-user-dropdown {
+        // Set the color of the dropdown arrow of the user dropdowm menu
+        color: $link-color !important;
+    }
+    .secondary {
+        @if variable-exists(customize-sign-in-btn) {
+            // Customize the sign-in button on the home page
+            a.sign-in-btn {
+                @include btn-styles($btn-sign-in-bg, $btn-sign-in-color, $btn-sign-in-border-color, $btn-sign-in-hover-bg, $btn-sign-in-hover-color, $btn-sign-in-hover-border-color);
+            }
+        }
+        @if variable-exists(customize-register-btn) {
+            // Customize the register button on the home page
+            a.register-btn {
+                @include btn-styles($btn-register-bg, $btn-register-color, $btn-register-border-color, $btn-register-hover-bg, $btn-register-hover-color, $btn-register-hover-border-color);
+            }
+        }
+    }
 
-  .nav-links .main .nav-tab a {
-      color: $link-color;
-  }
-  .nav-links .main .nav-tab a.active,
-  .nav-links .main .nav-tab a:hover {
-    border-bottom-color: $link-color;
-    border-bottom-style: solid;
-  }
+}
 
-  span {
-    color: $header-font-color; // Header text color
-  }
+.login-register {
+    .form-field .field-link {
+        // Customize the color of the links in the login/registration form.
+        color: $link-color;
+    }
+    .action-primary {
+        @if variable-exists(customize-logistration-action-btn) {
+            // Customize the sign-in/registration button on the login page
+            &.login-button, &.register-button {
+                @include btn-styles($btn-logistration-bg, $btn-logistration-color, $btn-logistration-border-color, $btn-logistration-hover-bg, $btn-logistration-hover-color, $btn-logistration-hover-border-color);
+                box-shadow: none;
+            }
+        }
+    }
+}
+
+@if variable-exists(login-register-header-color) {
+    .login-register-content h2 {
+        // Customize the color of the login/registration page header
+        color: $login-register-header-color;
+    }
+}
 
-  .nav-links .secondary {
-    .nav-item a {
-      color: $link-color;
+.courses-container .courses .course {
+    @if variable-exists(accent-color) {
+        // Customize the bottom border color on hover for the course card in the home page
+        border-bottom-color: $accent-color;
     }
-    .dropdown-user-menu .dropdown-item a {
-      color: $link-color;
+    .course-image .cover-image .learn-more {
+        // Set the color of the 'Learn more' button in the course card in the home page
+        background-color: $btn-secondary-bg;
+        color: $btn-secondary-color;
     }
-  }
 }
 
-.wrapper-footer {
-  background: $footer-bg; // Footer color
-  background-color: $footer-bg !important; // Footer color
+.home>header .title>.heading-group {
```

Comment:
Can you add a setting to hide the hero from the page?

Reply:
@giovannicimolin, I can do it, but since there is no client instance using it at the moment, I didn't. Since it is just a one-line change we could always add it later when needed. What do you say?

Reply:
Looks good to leave as is, let's wait for a client to ask for this to implement it :+1: